int32 id
string name
string surname
string address
bool to_deliver
